packages/create-playwright/assets/example.spec.ts

```diff
@@ -1,7 +1,89 @@
-import { test, expect } from '@playwright/test';
+import { expect, Page, test as base } from '@playwright/test';
```

Comment:
- What happens to example.spec.js?
- I don't think this works as a "get started example". I'd rather make "1-get-started", "2-recipes", "3-pom" directories with examples or something like that.

Reply:
Will follow-up in more PRs dedicated to example test files.